### PR TITLE
Consistency between hosted agents for holo

### PR DESCRIPTION
**Pull request by zo-el**
### Change Log:

- Ability to create object of `hosted-agent` (aka Grandma) running on live `Hosts`
```js
     const alice: HostedPlayer = await s.hostedPlayers( {
        id: 'holofuel', // hosted agent instance_id
        agent_address: 'HcScivWRCRMeky9xa7k87tpuF5wnEzy5hOUUTphyIa5kw4i7s5dXyJ7ddrxyahz', //hosted agent address
        dna_address: "",
        host_id: '2zwc1vwrjav2199fwmrmirbyyhlj6hyxmkn1m0rojz98c259gq', // test host #1 uri
        host_email: 'joel+hpos1@holo.host', // test host #1 email
        host_password: 'asdfasdf' // test host #1 pwd
      } )

```

- Update `simpleConsistency()` to take in hosted-agent objects 
```js
    s.simpleConsistency(instance_id: string, players: Array<Player>, hosted_players: Array<HostedPlayer>)
```

### CODE REVIEW

File: test/e2e/test-dynamic-on.ts [hunk 1]
```diff
@@ -123,4 +123,47 @@ module.exports = (testOrchestrator, testConfig) => {
     t.end()
   })
 
+  test('test with hostedPlayers instances and run consistency', async t => {
```
Comment (zippy) [inline test/e2e/test-dynamic-on.ts:126]: This test doesn't include a zome-call through to the hosted players?

**Review by zippy — CHANGES_REQUESTED** [review]
This all looks great, but I think we really also want to ensure that zome calls can be made to hosted players, not just admin calls.  And it looks like the regular zome call function that's available on Player isn't available on HostedPlayer.

**Comment by maackle** [comment]
can this be closed?

**Comment by jost-s** [comment]
@zo-el This looks out of date. Can it be closed?

**Pull request closed without merging**